documentation/aircraft-commands.md

```diff
@@ -276,6 +277,13 @@ Altitude also accepts an `expedite` or `x` argument which can be used as the las
 
 _Syntax -_ `AAL123 c [alt]` or `AAL123 c [alt] x`
 
+## Fly Present Heading
+_Aliases -_ `fph`
+
+_Information -_ This command has the aircraft fly at the present heading, disregarding the passing of any waypoints. 
```

Comment:
The phrasing is a bit strange, could you make it clearer?

Reply:
Maybe to the effect of "fly straight ahead, regardless of flight plan routing" or something?

Reply:
done